vpr/src/device/rr_graph_builder.h

```diff
@@ -208,6 +227,8 @@ class RRGraphBuilder {
     t_rr_graph_storage& node_storage_;
     /* Fast look-up for rr nodes */
     RRSpatialLookup node_lookup_;
+    MetadataStorage<int>& rr_node_metadata_;
```

Comment:
Add comment:
Metadata is extra data on rr-nodes and edges, respectively, that is not used by vpr but simply passed through the flow so that it can be used by downstream tools. The main (perhaps only) current use of this metadata is the fasm tool of symbiflow, which needs extra metadata on which programming bits control which switch in order to produce a bitstream.

Reply:
Comment added